Module not found error when using ReactMemoComponent

Using `ReactMemoComponent` causes an error.

```
Module not found: Error: Can't resolve 'React' in '/home/oliver/repos/moto2/src/Client/output'
```

I think it's because the case, the import should be lowercase `react`.

--- a/Feliz.CompilerPlugins/ReactComponent.fs
+++ b/Feliz.CompilerPlugins/ReactComponent.fs
@@ -87,7 +87,7 @@ type ReactComponentAttribute(?exportDefault: bool, ?import: string, ?from:string
                     { decl with Body = AstUtils.emptyReactElement reactElType }
 
                 elif memo = Some true then
-                    let memoFn = AstUtils.makeImport "memo" "React"
+                    let memoFn = AstUtils.makeImport "memo" "react"
                     let info = AstUtils.MemberInfo(decl.Info, isValue=true)
                     let body = Fable.Delegate(decl.Args, decl.Body, None)
                     { decl with Info = info; Args = []; Body = AstUtils.makeCall memoFn [body] }

